browser-app/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "private": true,
+  "name": "browser-app",
+  "version": "0.0.0",
+  "dependencies": {
+    "@theia/core": "latest",
```

Comment:
Could you use 0.3.10 version everywhere, to prevent potential breaking browser-app and electron-app modules, after newer Theia realeses?